azure/packages/azure-service-utils/package.json

```diff
@@ -51,11 +51,12 @@
 		"api-extractor:commonjs": "flub generate entrypoints --outDir ./dist",
 		"api-extractor:esnext": "flub generate entrypoints --outDir ./lib --node10TypeCompat",
 		"build": "fluid-build . --task build",
+		"build:api-reports": "concurrently \"npm:build:api-reports:*\"",
+		"build:api-reports:current": "api-extractor run --local --config api-extractor/api-extractor.current.json",
+		"build:api-reports:legacy": "api-extractor run --local --config api-extractor/api-extractor.legacy.json",
 		"build:commonjs": "fluid-build . --task commonjs",
 		"build:compile": "fluid-build . --task compile",
-		"build:docs": "concurrently \"npm:build:docs:*\"",
-		"build:docs:current": "api-extractor run --local",
-		"build:docs:legacy": "api-extractor run --local --config api-extractor/api-extractor.legacy.json",
+		"build:docs": "api-extractor run --local",
```

Comment:
Not for this PR, but I would like to rename this script. I think the name is misleading. This is generating a docs model, not actually building any documentation. Ideally we could reclaim this for things like `build:readme` and other actual documentation generation.

Reply:
👍 The `api-extractor:*` script entries also need to be renamed.

Reply:
...I swear that wasn't there when I read it..... I am suspicious......
![image](https://github.com/microsoft/FluidFramework/assets/54606601/55556193-f79f-400f-8fd0-8410cdc4290b)